packages/integrations/node/README.md

```diff
@@ -150,6 +150,10 @@ You can pass the path to your key and certification via the environment variable
 SERVER_KEY_PATH=./private/key.pem SERVER_CERT_PATH=./private/cert.pem node ./dist/server/entry.mjs
 ```
 
+#### Runtime environment variables
+
+If an `.env` file is present when the build process is run, the values of environment variables provided by this file will be hard-coded in the output, same as when generating a static website. Special handling is required for values that have to be provided at run-time. Such variables must exist with any value during the build process: `RUNTIME_VALUE=placeholder astro build`. The placeholder value will be ignored by the build process, however its existence will signal to Astro that the actual value will be available as an environment variable when the built application will be run. In the case of multiple variables, you can store them in a file like `.env.runtime` and then start the build with the following command: `export $(cat .env.runtime) && astro build`. Note that these variables **must be absent** in the `.env` file.
```

Comment:
> https://github.com/withastro/astro/pull/7378#discussion_r1233608066

After the first section, you explain how to use it with multiple variables. We wouldn't want to lead people down the wrong path before asking them to do it another way. We can probably still make this clear in the text with something like "you must provide Astro with every environment variable".

In the final section, we shouldn't need to mention it's convenience, because we're trying to teach the user the one, easy way to do something.

```suggestion
If an `.env` file containing environment variables is present when the build process is run, these values will be hard-coded in the output, just as when generating a static website.

During the build, the variable must be absent from the `.env` file, and you must provide Astro with every environment variable to expect at run-time: `VARIABLE_1=placeholder astro build`. This signals to Astro that the actual value will be available when the built application is run. The placeholder value will be ignored by the build process, and Astro will use the value provided at run-time.

In the case of multiple run-time variables, store them in a seperate file (e.g. `.env.runtime`) from `.env`. Start the build with the following command:

```sh
export $(cat .env.runtime) && astro build
```
```